src/app/content/search/hooks/index.ts

```diff
@@ -77,6 +78,7 @@ export const receiveSearchHook: ActionHookBody<typeof receiveSearchResults> = (s
     ? {
       hash: selectedResult.result.source.elementId,
       search: queryString.stringify({
+        modal: persistentQueryParams.modal,
```

Comment:
@bethshook Fix working for me too, though I think you also want it for the other branch for when the query doesn't have results